cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  labels:
+    test.appstudio.openshift.io/optional: "false" # Change to "true" if you don't need the test to be mandatory
+  name: content-sources-backend-bonfire-tekton
+  namespace: rh-hms-tenant
+spec:
+  application: content-sources-backend
+  resolverRef:
+    params:
+    - name: url
+      value: https://github.com/RedHatInsigths/bonfire-tekton.git
+    - name: revision
+      value: main # Or whatever branch you want to test
+    - name: pathInRepo
+      value: pipelines/basic.yaml # This is the path to the pipeline
+    resolver: git
+  params:
+    - name: APP_NAME
+      value: content-sources # Name of app-sre "application" folder this component lives in.
+#    - name: COMPONENTS
+#      value: # Space-separated list of components to load.
+    - name: COMPONENTS_W_RESOURCES
+      value: pulp # List of components to keep.
+#    - name: BONFIRE_COMPONENT_NAME
+#      value: # Name of app-sre "resourceTemplate" in deploy.yaml for this component. If it is the same as the name in Konflux, you don't need to fill this
+    - name: COMPONENT_NAME
+      value: content-sources-backend # Name of your component name in Konflux
+    - name: IQE_PLUGINS
+      value: content-sources # Name of the IQE plugin for this app.
+    - name: IQE_MARKER_EXPRESSION
+      value: api # This is the value passed to pytest -m. Default is ""
+    - name: IQE_FILTER_EXPRESSION
+      value: not (introspection or rbac) # This is the value passed to pytest -k. Default is "" when no filter desired
```

Comment:
Maybe put in quotes?

Reply:
Yeah i thought so 
But then should other values also be in quotes ?

Reply:
If I remember correctly, by default yaml value is strings. So the single word values are fine. Something like `not (introspection or rbac)` I'd keep in quotes as safety thing :P

Reply:
cool, no problem! i will add that then.
Thanks !

Reply:
Asking if devs want to simply keep it as "not"

Reply:
You'd probably need it. But yeah, I'd say include Stephen into the discussion as well. It's more of a QE thing.

Reply:
yep just did thanks !